### PR TITLE
Spelling, grammar and minor clarifications.

### PR DESCRIPTION
**Pull request by Hasimir**
Spelling: weather/whether.

Grammar: slight adjustments.

New content: Changed Apache to Apache httpd (the name of their server,
differentiates from the hundreds of other projects they have),
introduced Nginx at the same time and then split the two
configurations.  Added a note that other software may be able to be
used, but indicated sysadmins familiar with those packages should
configure them (they should be able to work it out from the existing
examples anyway).

### CODE REVIEW

File: use/httpserver.rst [hunk 1]
```diff
@@ -24,20 +24,22 @@ The HTTP comes with a couple of additional variables:
 Using the server
 ================
 
-At the root of the server, you will find a list of the plugins that have a Web
-interface, and a link to them. Each plugin has its own subdirectory(ies).
+At the root of the server, you will find a list of the plugins that
+have a Web interface, and a link to them. Each plugin has one or more
```
Comment (Mikaela) [inline use/httpserver.rst]: Should web be spelled with small letter? It seems to have been written with capital letter before, but I am not sure if that is correct.

File: use/httpserver.rst [hunk 2]
```diff
@@ -24,20 +24,22 @@ The HTTP comes with a couple of additional variables:
...
+Here is an example of Apache httpd configuration (I didn't test it
+with the rewrite, please notify me whether it works or not):
 
 .. code-block:: apache
 
     <VirtualHost 0.0.0.0:80>
         ServerName stats.yourdomain.org
         <Location />
-                ProxyPass http://localhost:8080/webstats/
+                ProxyPass HTTP://localhost:8080/webstats/
```
Comment (ProgVal) [inline use/httpserver.rst]: I've never seen a scheme written in upper-case.
Any particular reason for using it?

Comment (ProgVal) [inline use/httpserver.rst]: Ok, got it, you used sed.
Please fix it.

File: use/httpserver.rst [hunk 3]
```diff
@@ -53,21 +55,28 @@ It's also possible to use Nginx in front of Supybot's HTTP server. Create a new
         # Note that your default server should specify these ports
         listen 80;
         listen [::]:80;
-        # If your default server also has https configured, uncomment
+        # If your default server also has HTTPs configured, uncomment
```
Comment (ProgVal) [inline use/httpserver.rst]: please write it HTTPS

File: use/httpserver.rst [hunk 4]
```diff
@@ -53,21 +55,28 @@ It's also possible to use Nginx in front of Supybot's HTTP server. Create a new
...
         # the following two listen lines to enable it for this vhost.
         #listen 443;
         #listen [::]:443;
         server_name stats.yourdomain.org;
 
     location / {
-        proxy_pass http://localhost:8080/;
+        proxy_pass HTTP://localhost:8080/;
```
Comment (ProgVal) [inline use/httpserver.rst]: same comment as above.

**Comment by Mikaela** [comment]
@ProgVal 

```
2015-01-27.log:[16:45:09] <Limnoria> Limnoria/Limnoria-doc: Hasimir opened pull request #63 (to master): Spelling, grammar and minor clarifications. https://github.com/Limnoria/Limnoria-doc/pull/63
2015-01-27.log-[16:46:06] <Mikaela> and I am here missing repo names and going to tell people that pull requests go to testing...
2015-01-27.log-[16:47:09] <Mikaela> there seem to be a lot previously unnoticed stupid mistakes which were most likely made by me
2015-01-27.log:[16:48:12] <Hasimir> there is no testing branch in -doc, but sure next time  :)
2015-01-27.log-[16:48:38] <Mikaela> pinkieval: we need you to create testing branch to -doc so we can open pull requests there! :)
2015-01-27.log:[16:49:04] <Mikaela> Hasimir: why are those HTTP's capitalized?
2015-01-27.log:[16:49:04] <Hasimir> a branch or a separate repo?
2015-01-27.log:[16:49:13] <Hasimir> because it's an acronym
2015-01-27.log-[16:49:14] <Mikaela> a branch
2015-01-27.log-[16:49:20] <Mikaela> but it would be useless so maybe not
2015-01-27.log-[16:49:39] <Mikaela> I see
2015-01-27.log:[16:49:58] <Hasimir> but the Apache httpd (and the index) are lower because the former is the way they've named it and the latter links somewhere
2015-01-27.log-[16:50:34] <Mikaela> I don't see anything else than https://github.com/Limnoria/Limnoria-doc/pull/63/files#r23612021
2015-01-27.log:[16:50:38] <Limnoria> Title: Spelling, grammar and minor clarifications. by Hasimir · Pull Request #63 · Limnoria/Limnoria-doc · GitHub (at github.com)
2015-01-27.log:[16:51:00] <Hasimir> it started with the "weather" bit
2015-01-27.log:[16:51:18] <Hasimir> the rest is just readability
2015-01-27.log:[16:51:50] <Hasimir> I use my usual quality control: do I wince when I read it?  if yes, fix, if not, it's fine
2015-01-27.log-[16:52:22] <Mikaela> I see
2015-01-27.log:[16:53:05] <Hasimir> though I try to ignore dialectic differences
2015-01-27.log:[16:53:19] <Hasimir> like American English ...
```

mainly

```
2015-01-27.log:[16:49:04] <Mikaela> Hasimir: why are those HTTP's capitalized?
2015-01-27.log:[16:49:13] <Hasimir> because it's an acronym
```

**Comment by GLolol** [comment]
Yes, HTTP is an acronym, but URIs are generally typed in lowercase (you don't see a browser ever say `HTTP://google.com`, do you?!)

**Comment by Hasimir** [comment]
Ah shoot, I should've spotted those bits, my bad.

Fixing it now.

**Comment by Hasimir** [comment]
Right, sorted.

Oh, it wasn't sed, but it was something equally as blind.  ;)

**Pull request merged**